From c8d2c332da2a03ed80d62628bbf54187a66f49b6 Mon Sep 17 00:00:00 2001
From: "Yann E. MORIN" <yann.morin.1998@free.fr>
Date: Fri, 24 Jul 2015 20:20:33 +0200
Subject: [PATCH] host-applications: disable missing applications

Those applications are missing from the public rpi-firmware package, so
do not even try to build them.

Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>

[Rebased for rpi-userland-a1b89e91f393c7134b4cdc36431f863bb3333163]
Signed-off-by: Peter Seiderer <ps.report@gmx.net>
---
 host_applications/linux/CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/host_applications/linux/CMakeLists.txt b/host_applications/linux/CMakeLists.txt
index 5a648e7..83bb66e 100644
--- a/host_applications/linux/CMakeLists.txt
+++ b/host_applications/linux/CMakeLists.txt
@@ -12,10 +12,7 @@ add_subdirectory(apps/dtoverlay)
 add_subdirectory(apps/dtmerge)
 
 if(ALL_APPS)
- add_subdirectory(apps/vcdbg)
- add_subdirectory(libs/elftoolchain)
  # add_subdirectory(apps/smct)
- add_subdirectory(apps/edid_parser)
  add_subdirectory(apps/hello_pi)
 endif()
 
-- 
2.8.1

